src/transformers/integrations/peft.py

```diff
@@ -350,7 +354,10 @@ def load_adapter(
                 # We only want to call prepare_model_for_compiled_hotswap once
                 self._prepare_peft_hotswap_kwargs = None
         else:
-            from peft.utils.hotswap import check_hotswap_configs_compatible, hotswap_adapter_from_state_dict
+            from peft.utils.hotswap import (
+                check_hotswap_configs_compatible,
+                hotswap_adapter_from_state_dict,
+            )
```

Comment:
Still a lot of unrelated style changes, sorry! Can you check against `main` and revert the unrelated changes that are just style like this? Most good IDEs have git compare/diff functions that should make it easy enough to do, but doing it manually shouldn't take too long either.